docs/TuxStores.md

```diff
@@ -0,0 +1,216 @@
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-Stores"></a>Requiring Stores [#](#Requiring-Stores)
+Tux uses Facebook's [Flux](https://facebook.github.io/flux/) architecture and provides an interface for creating stores. This interface is exposed via:
+
+```
+    var Stores = require('tux/Stores')
+```
+
+### <a id="Creating-Stores"></a>Creating Stores [#](#Creating-Stores)
+The `Stores` object allows the user to create stores via invoking [Stores.createStore](#createStores). Stores are responsible for managing data and alerting views to when that data has changed in TuxedoJS. Pass in an object to the `createStore` method and it will return a new object that has been extended with the `emitChange`, `addChangeListener`, and `removeChangeListener` methods that are common to Flux stores. The Tux methods are added to the object being extended first, so if you want to overwrite Tux's implementation of one or all of those methods, you just need to provide your methods with those keys on the input object to `createStore`. Let's take a look at an example: [#](#create-store-example)
+<a id="create-store-example"></a>
+
+```
+    var messageStore = Stores.createStore({
+      _messages: {},
+      getAll: function () {
+        return Object.keys(this._messages);
+      }
+    });
+```
+
+Below is the API documentation for this method.
+
+### <a id="Stores-createStore"></a>Stores.createStore [#](#Stores-createStore)
+
+```
+    var store = Stores.createStore(methods);
+```
+
+#### <a id="methods"></a>Parameter - `methods` - type: OBJECT - optional [#](#methods)
+Methods and properties to extend the created store with. This object is optional and there are no expected keys. However normally, as in the case of the example above, a store will have properties associated with storing its data and methods for exposing that data to view components.
+
+#### <a id="storeReturn"></a>Return - `store` - type: OBJECT [#](#storeReturn)
+The `Stores.createStore` method returns a `store` instance which is described below.
+
+#### <a id="store"></a>store - type: OBJECT [#](#store)
+The store has the following methods:
+
+### <a id="store-emitChange"></a>Method `store.emitChange` - type: FUNCTION [#](#store-emitChange)
+Normally, it is common to invoke `emitChange` after data in the store is updated to alert the views that are listening to this store to request the new data. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.emitChange(EVENT_TYPE); // OR
+    store.emitChange(); // will emit the default change event type string: 'CHANGE'
+```
+
+##### <a id="emitChange-EVENT_TYPE"></a>Parameter - `EVENT_TYPE` - type: STRING - optional [#](#emitChange-EVENT_TYPE)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `EVENT_TYPE` is specified the default value of `CHANGE` will be used.
```

Comment:
Change sentence to: Change event type string that will be broadcast by the store.  Will trigger callbacks that are listening for this event type on the store.